modules/install-central-operator-external-db.adoc

```diff
@@ -32,7 +32,7 @@ For more information about {product-title-short} databases, see the link:https:/
 [WARNING]
 ====
 * If you have installed the Operator in a different namespace, {ocp} shows the name of that namespace rather than `rhacs-operator`.
-* You must install the {product-title} `Central` custom resource in its own project and not in the `rhacs-operator` and `openshift-operator` projects, or in the project in which you have installed the {product-title} Operator.
+* Red{nbsp}Hat recommends installing the {product-title} `Central` custom resource in a dedicated project. Do not install it in the project where you have installed the {product-title} Operator. Additionally, avoid installing it in any projects with names that begin with `kube`, `openshift`, or `redhat`, and in the `istio-system` project.
```

Comment:
```suggestion
* Red{nbsp}Hat recommends installing the {product-title} `Central` custom resource in a dedicated project. Do not install it in the project where you have installed the {product-title} Operator. Additionally, do not install it in any projects with names that begin with `kube`, `openshift`, or `redhat`, and in the `istio-system` project.
```
I'm not a native speaker, but "avoid" sounds pretty weak to me. Like avoiding a shallow puddle. However trying one of these has zero chance of success, because the operator will outright reject it...